app-modules/concern/src/Providers/ConcernServiceProvider.php

```diff
@@ -34,30 +34,30 @@
 </COPYRIGHT>
 */
 
-namespace AdvisingApp\Alert\Providers;
+namespace AdvisingApp\Concern\Providers;
 
-use AdvisingApp\Alert\AlertPlugin;
-use AdvisingApp\Alert\Events\AlertCreated;
-use AdvisingApp\Alert\Histories\AlertHistory;
-use AdvisingApp\Alert\Listeners\NotifySubscribersOfAlertCreated;
-use AdvisingApp\Alert\Models\Alert;
+use AdvisingApp\Concern\ConcernPlugin;
+use AdvisingApp\Concern\Events\ConcernCreated;
+use AdvisingApp\Concern\Histories\ConcernHistory;
+use AdvisingApp\Concern\Listeners\NotifySubscribersOfConcernCreated;
+use AdvisingApp\Concern\Models\Concern;
 use Filament\Panel;
 use Illuminate\Database\Eloquent\Relations\Relation;
 use Illuminate\Support\Facades\Event;
 use Illuminate\Support\ServiceProvider;
 
-class AlertServiceProvider extends ServiceProvider
+class ConcernServiceProvider extends ServiceProvider
 {
-    public function register(): void
+    public function register()
     {
-        Panel::configureUsing(fn (Panel $panel) => ($panel->getId() !== 'admin') || $panel->plugin(new AlertPlugin()));
+        Panel::configureUsing(fn (Panel $panel) => $panel->getId() !== 'admin' || $panel->plugin(new ConcernPlugin()));
     }
 
     public function boot(): void
     {
         Relation::morphMap([
-            'alert' => Alert::class,
-            'alert_history' => AlertHistory::class,
+            'concern' => Concern::class,
+            'concern_history' => ConcernHistory::class,
```

Comment:
Unfortunately without large scale database update we can't change the morph mapping key. The class change is valid but let's keep the original morph keys.